Server/routes/authRoute.js

```diff
@@ -20,4 +21,7 @@ router.post("/recovery/request", recoveryRequestController);
 router.post("/recovery/validate", validateRecoveryTokenController);
 router.post("/recovery/reset/", resetPasswordController);
 
+//Delete routes
+router.delete("/delete/:userId", deleteUserController);
```

Comment:
This can just go into the Auth routes section as it is related to auth, best to keep them all together